lib/ReactViews/Tool.tsx

```diff
@@ -44,4 +47,48 @@ class Tool extends React.Component<PropsType> {
   }
 }
 
+interface ToolButtonProps extends ToolProps {
+  icon: string;
+}
+
+@observer
+export class ToolButton extends React.Component<ToolButtonProps> {
```

Comment:
Where is this used?

Reply:
Yep, so I made these changes for the place-editor tool.
You can find the usage here: https://github.com/TerriaJS/qld-digital-twin/blob/use-place-editor/lib/Views/UserInterface.jsx#L43

That tool unlike `DiffTool` is invoked from the Nav, hence the toggle behavior.

Reply:
Yeah these changes now makes a lot more sense with the place editor use!